app/models/event.rb

```diff
@@ -0,0 +1,36 @@
+class Event
```

Comment:
don't forget to test on the unit level! we should have tests for each class we create 🎉